### ISSUE TITLE
Support `LogitsPostProcessorConfig` in Triton Backend

### ISSUE THREAD

**Issue by devm777** [NVIDIA/TensorRT-LLM#6202]
https://github.com/NVIDIA/TensorRT-LLM/blob/98428f330e2f1d1b5606ca55ec4d30f0970dcab4/triton_backend/inflight_batcher_llm/src/model_instance_state.cc#L697

`LogitsPostProcessorConfig` is currently hardcoded to null. Please add support for logits so that we users can set it in config.pbtxt

If a maintainer can give pointers, I can implement this change and raise a PR.

Thank you.

**Comment by SimengLiu-nv** [comment]
Hi @devm777 , thank you for your interest in contribution! 
Reference for Log usage: [test_logits_post_processor](https://github.com/NVIDIA/TensorRT-LLM/blob/b7c8a672da7709dd8847e7861028168c661f9fda/tests/unittest/bindings/test_executor_bindings.py#L1854)
Reference for triton_backend implementation: you can check the code changes related to `decodingConfig`.

**Comment by karljang** [comment]
@devm777 , were you able to resolve your question regarding LogitsPostProcessorConfig? Just checking in to see if this issue is still relevant for you.

**Comment by devm777** [comment]
Hi @karljang, unfortunately there is still no way to specify `LogitsPostProcessorConfig` in Triton backend which makes it impossible to do things like token biasing. So this issue is still very much relevant

**Comment by harshal-96** [comment]
Hi @SimengLiu-nv, I picked this up and submitted #18507: named logits post-processors declared in the LLMAPI model.yaml (python import specs, not eval'ed lambdas) and selected per request via a new optional sampling_param_logits_post_processor_name input, routed into SamplingParams(logits_processor=...). Covered by CPU unit tests. Feedback welcome.
